README.md

```diff
@@ -7,6 +7,11 @@ For SMU software engineering, this is a template for frontend development.
 Author: [Junhao Shen](junhaos@smu.edu)
 ## Available Scripts
 
+Create a `.env` file and put the following in it:
+```
+REACT_APP_BACKEND_URL=http://localhost:9000
```

Comment:
Please take a look of src/config.ts, I have define a process env for backend url, the onlything You need to do Is just change 9005 to the number you use for your backend server. Tell David that I forget that I had add define env var in config.ts instead of define in a file. Which means .env file is not neccessary in frontend project.